esp-hal/src/dma/buffers.rs

```diff
@@ -12,7 +12,30 @@ pub struct Preparation {
     /// memory?)
     #[cfg_attr(not(esp32s3), allow(dead_code))]
     pub(super) block_size: Option<DmaBufBlkSize>,
-    // burst_mode, alignment, check_owner, etc.
+
+    /// Most DMA channels allow software to configure whether the hardware
```

Comment:
Please provide a one-liner summary as the first line, otherwise nightly clippy will be sad.